tests/Functional/ContentProjectRenderTest.php

```diff
@@ -181,6 +201,19 @@ public function testProjectRendering(): void {
     $this->assertCount(1, $values);
     $values[0]->hasLink('Example website');
 
+    // Assert documents file.
+    $file_wrapper = $project_details->find('css', 'div.ecl-file');
+    $file_row = $file_wrapper->find('css', '.ecl-file .ecl-file__container');
+    $file_title = $file_row->find('css', '.ecl-file__title');
+    $this->assertContains('Test document 2', $file_title->getText());
+    $file_info_language = $file_row->find('css', '.ecl-file__info div.ecl-file__language');
+    $this->assertContains('English', $file_info_language->getText());
+    $file_info_properties = $file_row->find('css', '.ecl-file__info div.ecl-file__meta');
+    $this->assertContains('KB - PDF)', $file_info_properties->getText());
+    $file_download_link = $file_row->find('css', '.ecl-file__download');
+    $this->assertContains('/document.pdf', $file_download_link->getAttribute('href'));
+    $this->assertContains('Download', $file_download_link->getText());
```

Comment:
I miss the file size from the assertions.

Reply:
done